components/raftstore-v2/src/operation/life.rs

```diff
@@ -298,6 +298,30 @@ impl Store {
                     forward_destroy_to_source_peer(ctx, &msg);
                     return;
                 }
+                if extra_msg.get_type() == ExtraMessageType::MsgAvailabilityRequest {
```

Comment:
I suggest to drop the message despite whether the peer is destroyed or not. Handling the message can be complicated and it's very unlikely to happen.